### ISSUE TITLE
pagination for ael03 fails with default values

### ISSUE THREAD

**Issue by Melkiades** [insightsengineering/rlistings#106]
Failing test can be found in https://github.com/insightsengineering/scda.test/pull/28
```
>   pg_lst <- paginate_listing(lst_res)
Error in paginate_listing(lst_res) : 
  Assertion on 'names(colwidths)' failed: Must be setequal to {'CPID','ASR','TRT01A','AEDECOD','Date_First','ASTDY','Duration','AESEV','Related','Outcome','Treated','Action','SERREAS'}, but has different type.
```

**Comment by edelarua** [comment]
@Melkiades this should be fixed by https://github.com/insightsengineering/formatters/issues/122, but other errors will then need to be resolved in the test for `paginate_listing` (can be fixed by setting other arguments, e.g. setting `lpp = 30`)

**Comment by gmbecker** [comment]
@shajoezhu this is why we need to realistic tests in the actual package tests, because formatters (as of now) still has a clean tst battery, and the failures are only happening in the third party package.

This removes the core protection that CI is intended to provide (ie PRs can't even get merged if they cause test failures), to the detriment of the project as a whole

**Comment by shajoezhu** [comment]
I would be more than happy to include these more realistic tests in rlistings and formatters, in fact I proposed this initially last year, and that was why I placed one of the listing test in rlistings if you remember.

**Comment by gmbecker** [comment]
Fixed in #109, @Melkiades please confirm and reopen if that test continues to fail
